libkbfs/folder_branch_ops.go

```diff
@@ -1176,6 +1176,16 @@ func (fbo *folderBranchOps) SetInitialHeadFromServer(
 		return fmt.Errorf("MD with revision=%d not initialized", md.Revision())
 	}
 
+	// Return early if the head is already set.  This avoids taking
```

Comment:
Wait, so it looks like this function is called *every* time we fetch the root node, i.e. we do an MD head fetch every time we fetch the root node? Not for this PR, but it seems like we should make that unnecessary, since this function only ever does anything when head is `nil`, i.e. the first time. Maybe add a TODO?